rmc/resources/grch37/rmc.py

```diff
@@ -0,0 +1,374 @@
+
+
+FREEZES = [1, 2]
+"""
+List of RMC/MPC data versions.
+"""
+
+CURRENT_FREEZE = 2
+"""
+Current RMC/MPC data version.
+"""
+
+
+# Original regional missense constraint resource files
+MUTATION_RATE_TABLE_PATH = f"{RESOURCE_PREFIX}/GRCh37/exac/mutation_rate_table.tsv"
+"""
+Path to TSV containing ExAC mutation rates.
+"""
+
+DIVERGENCE_SCORES_TSV_PATH = (
+    f"{RESOURCE_PREFIX}/GRCh37/exac/divsites_gencodev19_all_transcripts.txt"
+)
+"""
+Path to text file with divergence scores per transcript.
+"""
+
+divergence_scores = TableResource(
+    path=f"{RESOURCE_PREFIX}/GRCh37/exac/ht/div_scores.ht",
+    import_func=hl.import_table,
+    import_args={
+        "path": DIVERGENCE_SCORES_TSV_PATH,
+        "key": "transcript",
+        "min_partitions": 50,
+        "impute": True,
+    },
+)
+"""
+Table with divergence score between humans and macaques for each canonical transcript in Gencode v19.
+"""
+
+
+# RMC-related resources
+constraint_prep = VersionedTableResource(
+    default_version=CURRENT_FREEZE,
+    versions={
+        freeze: TableResource(
+            path=f"{MODEL_PREFIX}/{CURRENT_VERSION}/{freeze}/context_obs_exp_annot.ht"
+        )
+        for freeze in FREEZES
+    },
+)
+"""
+Context Table ready for RMC calculations.
+
+HT is annotated with observed and expected variant counts per base.
+"""
+
+one_break = VersionedTableResource(
+    default_version=CURRENT_FREEZE,
+    versions={
+        freeze: TableResource(
+            path=f"{CONSTRAINT_PREFIX}/{CURRENT_VERSION}/{freeze}/one_break.ht"
+        )
+        for freeze in FREEZES
+    },
+)
+"""
+Table containing transcripts with at least one break.
+
+Found when searching constraint_prep HT for transcripts for a single (first) break.
+"""
+
+not_one_break = VersionedTableResource(
+    default_version=CURRENT_FREEZE,
+    versions={
+        freeze: TableResource(
+            path=f"{CONSTRAINT_PREFIX}/{CURRENT_VERSION}/{freeze}/not_one_break.ht"
+        )
+        for freeze in FREEZES
+    },
+)
+"""
+Table containing transcripts without one significant break.
+
+Transcripts in this table will be processed to check for two simultaneous breaks.
+"""
+
+not_one_break_grouped = VersionedTableResource(
+    default_version=CURRENT_FREEZE,
+    versions={
+        freeze: TableResource(
+            path=f"{CONSTRAINT_PREFIX}/{CURRENT_VERSION}/{freeze}/not_one_break_grouped.ht"
+        )
+        for freeze in FREEZES
+    },
+)
+"""
+Not one break Table grouped by transcript with observed missense, expected missense, and positions collected into lists.
+
+Input to searching for simultaneous breaks.
+"""
+
+multiple_breaks = VersionedTableResource(
+    default_version=CURRENT_FREEZE,
+    versions={
+        freeze: TableResource(
+            path=f"{CONSTRAINT_PREFIX}/{CURRENT_VERSION}/{freeze}/multiple_breaks.ht"
+        )
+        for freeze in FREEZES
+    },
+)
+"""
+Table containing transcripts with multiple breaks.
+"""
+
+simul_break_under_threshold = (
```

Comment:
Should we change this to `simul_break_under_threshold_path` for clarity? And then similarly modify the description to say "path to"

Reply:
Was this addressed?

Reply:
oops!